src/components/Table/index.tsx

```diff
@@ -76,26 +77,45 @@ export const Table = ({ headData, bodyData, isClosed }: TableProps) => {
         >
           {bodyData.map(({ purchases, color, title }) =>
             purchases.map((purchase: any, index) => {
+              console.log(purchase)
               return (
                 <TableRow key={index} padding={isMobile ? '0 0 0 10px' : '0 0 0 20px'}>
                   <TableColumn>
                     <TokenPriceLabel color={color}>{title}</TokenPriceLabel>
                   </TableColumn>
-                  {Object.getOwnPropertyNames(purchase).map((element: any, _index) => {
-                    return (
-                      <TableColumn key={_index}>
-                        <TokenPriceLabel>{purchase[element]}</TokenPriceLabel>
-                      </TableColumn>
-                    )
-                  })}
+                  {Object.getOwnPropertyNames(purchase)
```

Comment:
Use `Object.entries` please. This is not readable.

Reply:
Object. entries give me an array of both key and value pairs. I just want the keys. Do you want me to use Object. keys instead? but that would give me the same problem as before in a previous PR.

Reply:
But you're using the values. If you don't need the keys, `Object.values` is another utility 

```
                      <TableColumn key={_index}>
                        <TokenPriceLabel>{purchase[element]}</TokenPriceLabel>
                      </TableColumn>
```